Common_Files/Physics.ipynb

```diff
@@ -10,19 +10,53 @@
     "\n",
-    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light prior to taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"[insert the topic of problem here]\" as the first step for development and validation of CFD simulation for the problem of < insert the title of the problem > :**\n",
+    "**In other words understanding physics and theory of problem before developing a CFD simulation can be thought of turning on a flash light prior to taking any step in a completely dark room. Hence, let's review the fundamental physics and theory behind the problem of \"2D Laminar Flow over a rectangular plate.\" as the first step for development and validation of CFD simulation for the problem of < insert the title of the problem > :**\n",
     "\n",
     "# General Recommended Structure for Writing Physics File\n",
     "\n",
-    "Below is the recommended general structure to develop the Physics file for the flow of interest. Please note that we are always open to new ideas and modifying the structure below. Thus, do not limit yourself to the following structure and feel free to implement your suggestion and changes. We can modify this structure based on new feedback.\n",
+    "<img src=\"./Images/fluid1.jpg\" width=\"500\">\n",
+    "\n",
+    "                                           Fig. 1 - The visuallization of flow over a rectangle.\n",
+    "\n",
+    "Conservation of momentum\n",
+    "\n",
+    "\n",
+    "The equation for conservation of momentum is used to obtain a general understanding about the interaction between the flow and the rectangle. There are two force components acting on the rectangular body. One force component is in the stream wise direction and is called the Drag Force. The other component is perpendicular to the flow direction and is called the Lift Force (i.e. flow tries to lift-up the body). Theoretically, because the angle of attack between the flow and the rectangle’s top and bottom region is zero degree, thus there is no lift force. Even if there is lift force from the top and bottom region of the plate, they would cancel out due to symmetrical condition. Therefore, the net lift force on the rectangle will always be equal to zero. However, there will be a significant drag force on the rectangle walls as a result of it's interaction with the flow. To obtain an estimate for this drag force acting on the rectangle, one can consider the conservation of momentum in stream wise direction. The general form of conservation of momentum equation applied to the defined control volume showed in Fig.2 is as follows:\n",
+    "\n",
```

Comment:
If you can include the definition of the CV in your sketch at the beginning, you can refer to that sketch here. Consider this.